site.yml.sample

```diff
@@ -5,6 +5,7 @@
   become: True
   roles:
   - ceph-mon
+  - ceph-agent
```

Comment:
Someone trying to get this installed will usually want to deploy it on a monitor node as it needs to talk to the rest of the cluster. It can certainly be deployed to other nodes though.